docker/daemon.go

```diff
@@ -264,6 +249,22 @@ func (cli *DaemonCli) CmdDaemon(args ...string) error {
 		"graphdriver": d.GraphDriver().String(),
 	}).Info("Docker daemon")
 
+	api := apiserver.New(serverConfig)
```

Comment:
@calavera is there any particular reason why this was moved down here? My only concern is that this code is sensitive and I know we shouldn't be moving it for no reason.